eng/common/templates/job/source-build.yml

```diff
@@ -48,11 +48,11 @@ jobs:
     pool:
       ${{ if eq(variables['System.TeamProject'], 'public') }}:
         name: $[replace(replace(eq(contains(coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranch'], 'refs/heads/main'), 'release'), 'true'), True, 'NetCore-Svc-Public' ), False, 'NetCore-Public')]
-        demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
```

Comment:
Anything under eng/common may get updated or overwritten with the next arcade DARC update. I assume this change is being made in arcade also.